package.json

```diff
@@ -2,13 +2,18 @@
   "name": "node-polyglot",
   "version": "2.3.0",
   "description": "Give your JavaScript the ability to speak many languages.",
-  "main": "index.js",
+  "main": "dist/polyglot.common-js.js",
+  "module": "dist/polyglot.es-modules.js",
```

Comment:
Use of this field in existing packages has poisoned the ecosystem and is the reason node itself can't use "module"; it's a very hostile thing for the ecosystem to use new package.json fields like this, and we won't be doing it. Only "main", until node ships support for something else unflagged.

Reply:
@ljharb this is the primary value of the pull request, to enable front end bundlers to work with polyglot. This isn't my proposal, it's the one put forth by rollup/webpack: https://github.com/rollup/rollup/wiki/pkg.module

If you're saying this is no-go then there's pretty much no point to including this PR unless you think the other changes are worth accepting.

Reply:
Yes, i think the module field is hugely toxic to the ecosystem, and i don’t think it’s worth supporting.

Reply:
Then fork I shall! ;) Thanks for your time, and best of luck.

Reply:
Go for it - but I'm still not sure why. There's nothing to tree-shake, so there's zero benefit from having this module as native ESM.

Reply:
I'm not using tree shake. 

I'm interested in using modules without having to shim them from commonjs. I understand that polluting `package.json` with a bunch of proposals is not a good thing, but I also don't think modules should be based on commonjs. The web is beginning to move away from that. All that has to happen is `package.module` field points at the es module variant. With that, 
* node/commonjs stuff continues to work as expected
* frontend code bundled with webpack continues to work
* frontend code bundled with rollup now works and is able to use `import ...` directly without shims or compatibilitiy layers

Reply:
I'm not sure what you mean by "shim them from commonjs", but a CJS loader for ESM should work just fine as part of a build process.

As for CJS, the JS ecosystem will never fully move away from it - interoperability, and mixed graphs, are the only way to move forward. When node ships ESM unflagged, that will be providing that mechanism - and it won't be a package.json field by default.

Reply:
> I'm not sure what you mean by "shim them from commonjs", but a CJS loader for ESM 

A cjs loader is the required code shim just to be able to use the es modules feature.


> the JS ecosystem will never fully move away from it

The node js ecosystem may never fully move away from it yet, but other systems already are. Browser support is rapidly improving for es modules, and there are backend runtimes that are beginning to support it too. deno is a potential successor (written by Ryan Dahl, who wrote the original node) and this uses modules natively:  https://github.com/denoland/deno

Reply:
lol, deno is barely usable and is based on the premise that the very reasons node is successful were bad ideas. It's not going anywhere.

Reply:
> It's not going anywhere.

Lol ok man. That's just your opinion.